woocommerce/woocommerce-ios
# Order Filters: apply filters to order list

As part of #5243 in this PR the filters will be applied to the order list, so we allow merchants to filter by order status and date range.

## Changes
- Removed `orders-load-all-2.json` (no more used. Most of the changes are related to this file)
- Updated tracking event passing also the `date_range` parameter.
- Implemented `computedStartDate` and `computedEndDate` that will return date ranges based on date filter applied.
- Updated `OrderListSyncActionUseCase` removing the dual fetch (because previously we had two tabs `All orders` and `Processing`)
- Implemented the way for refreshing orders when filters are applied.
- Updated/added unit tests.

## Testing
1. Open the order list tab.
2. The order list should be list all the orders without any filter applied.
3. Try to play applying new filters to the order list: status and date ranges.
4. You should be able to see all the orders filtered, and to paginate them.

Update release notes:
